src/natives/screenshot/win/Program.cs

```diff
@@ -174,7 +174,7 @@ struct WindowBorderOnScreenFlags {
                     wi.rcClient.right -= 1;
                 }
             }
-            else if(browser == "ie") {
+            else if(browser == "iexplore") {
```

Comment:
Thanks, [got it](https://github.com/DevExpress/testcafe-browser-natives/pull/65/files#diff-aa42c270f3c052133d6798bd323aa0b2L83)

> Maybe it's worth to rename browser to processName.
Imho, cause we have browser specialized package, why not.